WalletWasabi.Gui/Controls/WalletExplorer/CoinInfoTabView.xaml

```diff
@@ -41,7 +41,7 @@
           <controls:ExtendedTextBox Grid.Column="1" Classes="selectableTextBlock monospaceFont Transparent" Text="{Binding PubKey}" />
         </Grid>
         <Grid ColumnDefinitions="150, *">
-          <TextBlock Classes="monospaceFont" Text="Clusters:" />
+          <TextBlock Classes="monospaceFont" Text="Known By:" />
```

Comment:
I see that you sometimes you use `Known By` and sometimes `Known by` we should stick with one.

Reply:
I used `Known By` in those places where you have to enter inputs because in that case the error message is clear `Known By is required`; otherwise `Known by is required` looks more like a grammatical mistake. But I a not sure, I was expecting you find it and discuss it.

Reply:
How about displaying it with quotation marks `"Known by" is required`?

What about this comment?

> Clusters should be renamed here https://github.com/zkSNACKs/WalletWasabi/blob/master/WalletWasabi.Gui/Controls/WalletExplorer/CoinListView.xaml#L92

Reply:
Done.